parser.cpp

```diff
@@ -1286,6 +1295,11 @@ namespace Sass {
       return result;
     }
 
+    if (lex< ampersand >())
+    {
```

Comment:
in this case, this wouldn't be left over code.  This is actually necessary to resolve the parent selector expression.